src/index.css

```diff
@@ -36,7 +36,8 @@ button.btn-light:focus {
 }
 
 .card:hover {
-  background-color:#f8f9fa;
+  transition: .2s;
+  transform: scale(1.04);
```

Comment:
Nice find! :) The hover makes the card a bit jumpy -- perhaps use `scale(1.01);` instead?

Reply:
Do you mean jumpy because too large on hover or the way it jumps back on blur?

Reply:
Reduced scale a bit; we can always tweak this later.